### PR TITLE
Add .editorconfig

### PR DESCRIPTION
**Pull request by vxcamiloxv**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
+indent_size = unset
+
+[*.{json,css}]
+indent_size = 2
+
+[*.{pug,jade}]
```
Comment (strugee) [inline .editorconfig]: I'm not super familiar with EditorConfig syntax but I assume you are, and that this glob-like thing works?

Comment (strugee) [reply .editorconfig]: I just checked and it's literally on the homepage; I should stop being so lazy

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+indent_size = 2
+trim_trailing_whitespace = false
+insert_final_newline = false
```
Comment (strugee) [inline .editorconfig]: This should probably be `true`

Comment (vxcamiloxv) [reply .editorconfig]: done

**Pull request merged**